### ISSUE TITLE
Chromedriver lookup became broken in 2.53.1 (Ruby binding)

### ISSUE THREAD

**Issue by abotalov** [SeleniumHQ/selenium#2273]
Currently when starting tests, I get the following error:
```
Unable to find the chromedriver executable. Please download the server from http://chromedriver.storage.googleapis.com/index.html and place it somewhere on your PATH. More info at https://github.com/SeleniumHQ/selenium/wiki/ChromeDriver. (Selenium::WebDriver::Error::WebDriverError)
```

It seems to be because of the following change in selenium-webdriver 2.53.1 (inside Selenium::Webdriver::Platform#find_binary):
`exe = File.join(path, binary_name)` was changed into `exe = Dir.glob(File.join(path, binary_name)).first`.

Ruby binding
Ruby version: 1.9.3
OS:  Windows 7
Selenium Version:  2.53.1
Browser:  Chrome
Browser Version:  51.0.2704.84 m

**Comment by jorgevbo** [comment]
Hi

I found chromedriver filepath is in wrong format. It's using wrong window slash ("/" "\").
I printed path to binary and obtain:

C:\Ruby22-x64\selenium-chrome-driver/chromedriver.exe

If I replace string to C:/Ruby22-x64/selenium-chrome-driver/chromedriver.exe it's works!

**Comment by jorgevbo** [comment]
Wrong function is in selenium/rb/lib/selenium/webdriver/common/platform.rb at find_binary method

**Comment by kapoorlakshya** [comment]
I have the same issue with Firefox 47 (geckodriver.exe) and Chrome.

`Selenium::WebDriver::Error::WebDriverError Exception: Unable to find geckodriver. Please download the executable from https://github.com/mozilla/geckodriver/releases`

**Comment by titusfortner** [comment]
I missed that windows appends `.exe` to file paths when I made it a fuzzy locator, so this is broken on Windows.

**Comment by titusfortner** [comment]
Actually, before I release, can someone with Windows verify this works in irb?
```
['geckodriver*', 'chromedriver*'].each do |bin_name|
  ENV['PATH'].split(File::PATH_SEPARATOR).each do |path|
    exe = Dir.glob(File.join(path, bin_name)).first
    next unless exe
    puts exe if File.executable?(exe)
  end
end
```

I'm on vacation for 2 more weeks without access to my Windows VM. :)

**Comment by titusfortner** [comment]
Fixed & released in selenium-webdriver-2.53.2.gem
